packages/devextreme/js/ui/shared/accessibility.js

```diff
@@ -57,7 +57,7 @@ function processKeyDown(viewName, instance, event, action, $mainElement, execute
 
 export function saveFocusedElementInfo(target, instance) {
     const $target = $(target);
-    const ariaLabel = $target.attr('aria-label');
+    const ariaLabel = $target.attr('aria-label')?.replace(/"/g, '\\"');
```

Comment:
It is needed to move the escaping to the getActiveAccessibleElements method level. Because you repaired only one case. There is [the second usage here](https://github.com/DevExpress/DevExtreme/blob/396fa5aa36e35ce0eaf7633a5fef1e6b2b6ccdd5/packages/devextreme/js/ui/shared/accessibility.js#L161).

Reply:
If I add the escape character in **getActiveAccessibleElements** method, it will not fix the **ariaLabel** variable inside **saveFocusedElementInfo** method, because ariaLabel variable is being used in [**focusedElementInfo**.](https://github.com/DevExpress/DevExtreme/pull/28107/files/057905cae5d6ee2b1ee86d71c5c21324c61210f7#diff-517e37cd44e0fb7d7211dc2a3f1732cc7e4585e6257898b60b9e57cb9e1a6cc8R64)

Reply:
I will just add another escape character for the variable I mentioned..